apps/console/src/features/identity-providers/components/settings/authenticator-settings.tsx

```diff
@@ -446,6 +446,61 @@ export const AuthenticatorSettings: FunctionComponent<IdentityProviderSettingsPr
         );
     };
 
+    /**
+     * A predicate that checks whether a give federated authenticator
+     * is a default authenticator.
+     * @param auth
+     * @returns true if {@code auth.data.isDefault} is truthy
+     */
+    const _isDefaultAuthenticatorPredicate = (
```

Comment:
Does the dangling underscore mean `private` function here?
Since this is a component and since we don't invoke these functions from outside, I don't see a point of having the leading `_`.

Check other places as well.

Reply:
Yes that was the intention but yeah, as you said we don't need to denote with those with a underscore since its nested. This fix is applied in commit 32bfc11 and 0dc1440.